docs/develop/convert-javascript-to-typescript.md

```diff
@@ -98,138 +100,8 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
-
-            // Run the queued-up command, and return a promise to indicate task completion
-            return ctx.sync()
-                .then(function () {
-                    var highestRow = 0;
-                    var highestCol = 0;
-                    var highestValue = sourceRange.values[0][0];
-
-                    // Find the cell to highlight
-                    for (var i = 0; i < sourceRange.rowCount; i++) {
-                        for (var j = 0; j < sourceRange.columnCount; j++) {
-                            if (!isNaN(sourceRange.values[i][j]) && sourceRange.values[i][j] > highestValue) {
-                                highestRow = i;
-                                highestCol = j;
-                                highestValue = sourceRange.values[i][j];
-                            }
-                        }
-                    }
-
-                    cellToHighlight = sourceRange.getCell(highestRow, highestCol);
-                    sourceRange.worksheet.getUsedRange().format.fill.clear();
-                    sourceRange.worksheet.getUsedRange().format.font.bold = false;
-
-                    // Highlight the cell
-                    cellToHighlight.format.fill.color = "orange";
-                    cellToHighlight.format.font.bold = true;
-                })
-                .then(ctx.sync);
-        })
-        .catch(errorHandler);
-    }
-
-    function displaySelectedCells() {
-        Office.context.document.getSelectedDataAsync(
-            Office.CoercionType.Text,
-            null,
-            function (result) {
-                if (result.status === Office.AsyncResultStatus.Succeeded) {
-                    showNotification('The selected text is:', '"' + result.value + '"');
-                } else {
-                    showNotification('Error', result.error.message);
-                }
-            });
-    }
-
-    // Helper function for treating errors
-    function errorHandler(error) {
-        // Always be sure to catch any accumulated errors that bubble up from the Excel.run execution
-        showNotification("Error", error);
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
-    }
-
-    // Helper function for displaying notifications
-    function showNotification(header, content) {
-        $("#notification-header").text(header);
-        $("#notification-body").text(content);
-        messageBanner.showBanner();
-        messageBanner.toggleExpansion();
-    }
-})();
-```
-
 ## See also
 
 - [Promise implementation discussion on StackOverflow](https://stackoverflow.com/questions/44461312/office-addins-file-in-its-typescript-version-doesnt-work)
 - [Office Add-in samples on GitHub](https://github.com/OfficeDev/Office-Add-in-samples)
+- [](../quickstarts/excel-quickstart-jquery.md)
```

Comment:
```suggestion
- [Build an Excel task pane add-in](../quickstarts/excel-quickstart-jquery.md)
```
Also, should this link to the Visual Studio version? Or, since the yo office link at the beginning was removed, maybe ink to the yo office version and have the link text reflect a VS Code version.

Reply:
There was originally a note that linked to this, but I'm not sure this even makes sense since this article walks you through creating a VS Office add-in. So I removed the link.